src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Aero2/PresentationFramework.Aero2_dacxb2zy_wpftmp.csproj

```diff
@@ -0,0 +1,107 @@
+<!-- PresentationFramework.Aero2 -->
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp5.0</TargetFramework>
```

Comment:
I think this should be `net5.0`.

Reply:
This is a generated file I didn't mean to commit. I'll remove it.